server/build.gradle

```diff
@@ -314,6 +314,10 @@ tasks.named('splitPackagesAudit').configure {
     'org.elasticsearch.cli.EnvironmentAwareCommand',
     'org.elasticsearch.cli.KeyStoreAwareCommand',
     'org.elasticsearch.cli.LoggingAwareCommand',
-    'org.elasticsearch.cli.LoggingAwareMultiCommand'
+    'org.elasticsearch.cli.LoggingAwareMultiCommand',
+
+    // We temporarily need to have modified versions of the lucene Tessellator and LatLonShape classes while we wait for a bugfix
+    'org.apache.lucene.document.XLatLonShape',
+    'org.apache.lucene.geo.XTessellator'
```

Comment:
Then we loose access to parts of Lucene that are not public, but package-private. Using the same package name was, in fact, your idea when we discovered this issue last week. I was in the process of copying more and more classes over, not sure where it would end when you suggested this (ie. keep package name, and change class name instead).

If you want me to try the other approach, I can give it a shot, but suspect it might not work (ie. we might need to copy too much of Lucene).